node/avn-service/src/ethereum_events_handler.rs

```diff
@@ -0,0 +1,129 @@
+use sp_avn_common::event_types::{
+    AddedValidatorData, AvtGrowthLiftedData, Error, EthEvent, EthEventId, EventData, LiftedData,
+    NftCancelListingData, NftEndBatchListingData, NftMintData, NftTransferToData, ValidEvents,
+};
```

Comment:
regarding sp_avn_common::Error and expanding it with these additional use cases.
I think it would be cleaner, if we had here its own Error module and map any errors into equivalent ones here.
For example all parsing errors can be ParsingError.
It would also be beneficial if we had some error/info/trace logs here to help with future investigations, especially with the interactions with Ethereum as this has been a pain point in the past